net/yggdrasil-jumper/Makefile

```diff
@@ -0,0 +1,48 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=yggdrasil-jumper
+PKG_VERSION:=0.3.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/one-d-wide/yggdrasil-jumper/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=d0fd59e9f7a110094c9189b49b812fb0014c6c45535b4d30effd7cf602961454
+
+PKG_MAINTAINER:=one-d-wide <one-d-wide@protonmail.com>
```

Comment:
You need to have here real name, otherwise it wont be accepted.

Reply:
Do we have a field for `PKG_MAINTAINER_TAG`? Finding maintainers via their github tag (username) is a drag sometimes. Or perhaps a way of squeezing it into the same field above? Sorry to hijack...

Reply:
There isn't seems to be a special variable used for maintainer's github tag anywhere in this repository.
I could add link to by github page as a comment next to `PKG_MAINTAINER`, but it seems not to be a common practice.

Reply:
Hey @systemcrash , why do you need to have ``PKG_MAINTAINER_TAG``? Its pointless, because you or someone else should be more strict while merging commits. 

1. The tag applies only for GitHub, right. What about other mirrors? Don't forget that OpenWrt uses primary their own Git instance.

2. Username can be changed anytime.

3. Usually, if the committer email is not linked with the GitHub, I asked them politely to do so, because then the real name is linked to the GitHub account and you can mention that anytime and you can find it.

4. While merging commits, which are done by PR, there is always history (in Git or even here at GitHub) and referenced PR, which can be found at anytime. You can reach contributor via email and now we can have discussion what we will do if the email is no longer used, haha! 😆 

No need to have variables for everything, when they should be rules. :) Dont forget about that.

Reply:
Heh. OK, with #23084 merged, we can close this discussion.